policies/release-requirements.md

```diff
@@ -0,0 +1,137 @@
+  are closed.
+- There are no outstanding untriaged Coverity issues.
+- Coveralls coverage has not decreased overall from the previous release.
+- The CI must pass on the tip of the development branch before the release
+  commits are added to the tree, including the daily CI builds.
+- The tree must be frozen for at least 3 business days. No changes apart from
+  regression or security fixes should be merged during the freeze.
+- For 2 days before the release there should be no changes to ensure the daily
+  CI builds run on the development tree tip.
+- In case of the first beta release the OTC should explicitly approve
+  that the source is ready for a release with a vote.
+
+Major and Minor Releases
+------------------------
+
+As the release comes after the beta releases there is no need to repeat the
+stability requirements as those should be held already by the beta releases.
+
+- All issues [triaged] as regressions on the development branch from which the
+  release is to be done must have a milestone assigned.
+  _Regressions are considered from the previous stable release series._
+- All issues or pull requests with the milestone for the release are closed.
+- There are no outstanding untriaged Coverity issues.
+- Coveralls coverage has not decreased overall from the previous release from
+  the particular development branch.
+- The CI must pass on the tip of the development branch before the release
+  commits are added to the tree, including the daily CI builds.
+- The tree must be frozen for at least 7 days. No changes apart from regression
+  or security fixes should be merged during the freeze.
+- For 2 days before the release there should be no changes to ensure the daily
+  CI builds run on the development tree tip.
+- The OTC should explicitly approve that the source is ready for a release with
+  a vote.
+
+Patch Releases
+--------------
+
+The patch releases follow a similar process to major and minor releases with
+some simplifications as they are much more frequent and the tree stability
+requirements for the stable development branches should ensure there is
+a minimum amount of regressions in between the patch releases.
+
+- All issues [triaged] as regressions on the development branch from which the
+  release is to be done must have a milestone assigned.
+  _Regressions are considered from the previous minor, major or patch release
+  from the development branch._
+- All issues or pull requests with the milestone for the release are closed.
+- The CI must pass on the tip of the development branch before the release
+  commits are added to the tree, including the daily CI builds.
+- The tree must be frozen for at least 3 business days. No changes apart from
+  regression or security fixes should be merged during the freeze.
+- For 2 days before the release there should be no changes to ensure the daily
+  CI builds run on the development tree tip.
+- Embargoed security fixes are excepted from the rule above as they cannot
```

Comment:
Are there more steps associated with security patches (CVE's and notifications)?

Reply:
IMO they should not belong here as they would confuse things. If we have any security release process policy, it should be there.

Reply:
it still sounds like Release requirements to me..

Reply:
I beg to differ.

Reply:
In order to do the security release these are requirements - It could be a link to another document...

Reply:
We can/should do that later.